src/Tools/dotnet-monitor/CollectionRules/Options/Actions/CollectTraceOptionsDefaults.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
+{
+    internal static class CollectTraceOptionsDefaults
+    {
+        public const int MetricsIntervalSeconds = 1;
+        public const bool RequestRundown = true;
+        public const int BufferSizeMegabytes = 256;
```

Comment:
This seems like a big constant given our recommended container limits. Does this buffer not end up in our memory?

Reply:
This is the existing default when doing a trace, but I agree. I think we need to adjust these downward a bit.

The buffer ends up in the memory of the target application. It's the size of the buffer that events are put into just before the diagnostic server pulls them out and sends them over the event pipe. If an event source cannot put a new event into the buffer due to running out of room, the event is dropped relatively silently. This makes code that is listening for a start and stop event interesting because it has to guess whether the stop event just hasn't come through yet or was dropped.